modules/monitoring-installing-cluster-observability-operator-using-the-web-console.adoc

```diff
@@ -19,10 +19,10 @@ Install the {coo-first} from OperatorHub by using the {product-title} web consol
 . Click  *{coo-full}* in the list of results.
 . Read the information about the Operator, and review the following default installation settings:
 +
-* *Update channel* -> *development*
-* *Version* -> <most_recent_version>
+* *Update channel* -> *stable*
+* *Version* -> *1.0.0* or later
 * *Installation mode* -> *All namespaces on the cluster (default)*
-* *Installed Namespace* -> *openshift-operators*
+* *Installed Namespace* -> *Operator recommended Namespace: openshift-cluster-observability-operator*
```

Comment:
Better mention
Check Enable Operator recommended cluster monitoring on this Namespace